docs/source/WritingTests.rst

```diff
@@ -538,6 +538,11 @@ Detailing the ``fetch_asset()`` attributes:
   file, including the file name. The first success will skip the next
   locations. Notice that for ``file://`` we just create a symbolic link in the
   cache directory, pointing to the file original location.
+* ``expire:`` (optional) time period that the cached file will be considered
+  valid. After that period, the file will be dowloaded again. The value can
+  be an integer greater than 0 or a string containig the time and the unit.
+  Example: 10d (ten days). Valid units are s (second), m (minute), h (hour)
+  and d (day).
```

Comment:
Ok, I'm not going to argue for 0.000001s :)
So, my current approach is to use `0` as *always expire* and negative as *don't expire* (the same as in `expire=None`). But according to your comment, you think `0` and negative should both expire the cache, right? I have no strong opinion here... I will change the code to follow your suggestion.